CHANGELOG.md

```diff
@@ -3,10 +3,40 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
 ## [master](https://github.com/cucumber/cucumber-rails/compare/v1.6.0...master) (Not yet released)
 There are currently no unreleased changes.
 
+## [v1.7.0](https://github.com/cucumber/cucumber-rails/compare/v1.6.0...v1.7.0) (2019-04-17)
+
+### Changed
+
+ * Drop rails `4.0` / `4.1` support ([#392](https://github.com/cucumber/cucumber-rails/pull/392) [deivid-rodriguez](https://github.com/deivid-rodriguez))
+ 
+ * Allow Ruby `2.6` users to `ERB.new` ([#399](https://github.com/cucumber/cucumber-rails/pull/399) [koic](https://github.com/koic))
+ 
```

Comment:
One more thing: These extra blank lines lead to different rendering of the markdown, so I think you should leave them out like in the older entries.

Reply:
I've checked the new MD and it looks fine. I think they're meant to be this way

Reply:
Hm, doesn't it add extra space for you when you look at https://github.com/cucumber/cucumber-rails/blob/66d1eb8826509647b03e3707659eb80d30efd473/CHANGELOG.md?

Reply:
Yes it does, I couldn't see it though for some reason... I'll fix this.